addon/lib/metrics.js

```diff
@@ -143,31 +161,34 @@ const Metrics = module.exports = {
     const dataParsed = variantMaker.parseTests(JSON.parse(data));
 
     store.experimentVariants[subject] = dataParsed;
+    Metrics.experimentFeaturesChanged(subject, dataParsed);
     Events.emit(EVENT_SEND_VARIANTS, {
       data: JSON.stringify(dataParsed),
       subject: self.id
     });
   },
 
   onExperimentPing: function(ev) {
-    const timestamp = Date.now();
     const { subject, data } = ev;
+    const dataParsed = JSON.parse(data);
```

Comment:
Do we need to wrap `JSON.parse()` in a `try..catch` in case it receives invalid `data`?